### PR TITLE
chore: walk type CST nodes

### PR DESCRIPTION
**Pull request by KFlash**

### CODE REVIEW

**Comment by KFlash** [comment]
@aladdin-add See #110

File: src/forEachChild.ts [hunk 1]
```diff
@@ -43,6 +43,11 @@ export function forEachChild(node: any, visitor: any): any {
     case SyntaxKind.CoverInitializedName:
       return visitNode(node.left, node, visitor) || visitNode(node.right, node, visitor);
     case SyntaxKind.BindingElement:
+      return (
+        visitNode(node.left, node, visitor) ||
+        visitNode(node.type, node, visitor) ||
+        visitNode(node.right, node, visitor)
```
Comment (aladdin-add) [inline src/forEachChild.ts:49]: why using `||`? IMO, all the nodes need to be visited.

**Review by aladdin-add — CHANGES_REQUESTED** [review]
3 duplicated cases found by lgtm. :)

File: src/forEachChild.ts (hunk 1 above)
Comment (KFlash) [reply src/forEachChild.ts:49]: Same as what Typescript does in their walker.

**Comment by KFlash** [comment]
I already fixed the lgtm issue :)

**Comment by KFlash** [comment]
@aladdin-add I figured out why the walker isn't working. Working on a fix!

**Comment by KFlash** [comment]
@aladdin-add I'm merging this because I can confirm it now walks. You forgot to add a 'forEachChild' in your 'visitor.ts' file.

**Pull request merged**